allwinner/address_test.go

```diff
@@ -0,0 +1,85 @@
+package allwinner
+
+import (
+	"os"
+	"path"
+	"testing"
+)
+
+func createDirs(root string, dirs ...string) string {
+	for _, dir := range dirs {
+		if err := os.MkdirAll(path.Join(root, dir), os.ModePerm); err != nil {
+			panic(err)
+		}
+	}
+	return root
+}
+
+func createFiles(root string, paths ...string) string {
+	for _, path_ := range paths {
+		if file, err := os.Create(path.Join(root, path_)); err != nil {
+			panic(err)
+		} else {
+			file.Close()
+		}
+	}
+	return root
+}
+
+func createSymLink(root string, source string, destination string) {
+
+	if err := os.Symlink(path.Join(root, source), path.Join(root, destination)); err != nil {
+		panic(err)
+	}
+}
+
+func Test_getDefaultBaseAddress_default(t *testing.T) {
```

Comment:
https://pkg.go.dev/testing says it should be named TestGetDefaultBaseAddress_default